build.sbt

```diff
@@ -30,5 +30,28 @@ lazy val root = (project in file(".")).
       "localhost",
       artifactoryUserName.getOrElse("noUsernameSet"),
       artifactoryPassword.getOrElse("noPasswordSet")
-    )
+    ),
+    initialCommands in console := """
```

Comment:
It's really just so that you have some things to work with in the repl (there's a bug where defining queryable instances doesn't work in the repl)